README.md

```diff
@@ -10,18 +10,20 @@ In order to complete this tutorial, you'll need to have some stuff installed on
 
 # Step 1
 ## Initializing the project
-
-Django contains a lot of tools and commands that help you quickly create web-apps. To start our project off, in your terminal please type
+Django contains a lot of tools and commands that help you quickly create web-apps. To start our project off, in your terminal please type:
 ```bash
-$ django-admin startproject django101
+$ cd djangogirls/
```

Comment:
Ja, dit is een beetje irritant, maar dit moet omdat iedereen die tutorial van Django-girls gevolgd heeft.

Daar maak je die map aan dus we moeten dat eigenlijk wel overnemen denk ik :sweat_smile: